packages/core/strapi/lib/Strapi.js

```diff
@@ -367,6 +371,9 @@ class Strapi {
 
     await this.runLifecyclesFunctions(LIFECYCLES.BOOTSTRAP);
 
+    const cronTasks = this.config.get('server.cron.tasks');
```

Comment:
I would actually register the crons before bootstrap and then do a cronRunner.start()
So you can add crons programmatically in the register. without them starting. and then your start everything.